charts/okd/airflow/values.yaml

```
@@ -45,13 +45,19 @@ images:
     pullPolicy: Always
 
 migrateDatabaseJob:
+  env:
+    - name: AIRFLOW__CORE__DAGS_ARE_PAUSED_AT_CREATION
+      value: "True"
```

These values should be `False` else they will be paused
```suggestion
      value: "False"
```